middle_end/flambda/simplify/simplify_variadic_primitive.ml

```diff
@@ -72,6 +72,51 @@ let simplify_make_block_of_values dacc _prim dbg tag ~shape
     Simplified_named.reachable term, env_extension, args, dacc
   end
 
+let simplify_make_block_of_floats dacc _prim dbg
+      ~(mutable_or_immutable : Mutability.t)
+      args_with_tys ~result_var =
+  let denv = DA.denv dacc in
+  let args, _arg_tys = List.split args_with_tys in
```

Comment:
There doesn't seem any point in allocating `_arg_tys`, I suggest using `List.map fst args_with_tys` instead.

Reply:
done